frontend/lib/framework/error-classes-list.cpp

```diff
@@ -128,6 +129,20 @@ void ErrorMissingInclude::write(ErrorWriterBase& wr) const {
   wr.note(moduleInclude, "expected file at path '", filePath, "'");
 }
 
+void ErrorUseImportUnknown::write(ErrorWriterBase& wr) const {
+  auto visibilityClause = std::get<const uast::VisibilityClause*>(info);
+  auto symbolName = std::get<std::string>(info);
+  auto searchedScope = std::get<const resolution::Scope*>(info);
+  auto useOrImport = std::get<const resolution::VisibilityStmtKind>(info);
+  wr.heading(kind_, type_, locationOnly(visibilityClause),
```

Comment:
I'm reviewing commit-by-commit, so sorry if this is addressed later, but this one needs a `.` at the end of the sentence. it's auto-stripped in brief mode.